src/components/widgets/SelectWidget.js

```diff
@@ -32,22 +32,13 @@ function getValue(event, multiple) {
   }
 }
 
-function getDefaultValue(value, multiple, options, disabledOptions) {
-  if (multiple) {
-    let filter = options.filter(o => {
-      if (disabledOptions) {
-        return (
-          disabledOptions.indexOf(o.value) === -1 &&
-          value.indexOf(o.value) !== -1
-        );
-      } else {
-        return value.indexOf(o.value) !== -1;
-      }
-    });
-
-    return filter;
-  } else {
-    return value;
+function makeSelectedValue(value = "", options) {
```

Comment:
@Inam-urRehman did you verified this logic with following:
- Normal select
- Select with disabled fields
- Multi Select
- Multi Select with Disabled Fields

Reply:
Normal and multi select has been tested. Disabled things is not verified from my side